src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/YourSingleApprenticeReadyForReviewStatus.cshtml

```diff
@@ -0,0 +1,66 @@
+﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+@using SFA.DAS.EmployerAccounts.Web.Helpers
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.CohortV2ViewModel
+
+<section class="dashboard-section">
+    <h2 class="section-heading heading-large" id="call-to-action-your-apprentice-status">
+        Your apprentice
+    </h2>
+
+    <table class="responsive">
+        <tr>
+            <th scope="row" class="tw-35">
+                Name
+            </th>
+            <td class="tw-65" id="apprentice-name">
+                @(Model?.Apprenticeships?.First().ApprenticeshipFullName)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Training Provider
+            </th>
+            <td id="provider-name">
+                @(Model?.Apprenticeships?.First().TrainingProvider?.Name)
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Course
+            </th>
+            <td id="course-name">
+                @(Model?.Apprenticeships?.First().CourseName)                
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Status
+            </th>
+            <td id="ready-for-review" class="govuk-tag govuk-tag--error">
+                READY FOR REVIEW
+            </td>
+        </tr>
+        <tr>
+            <th scope="row">
+                Start month
+            </th>
+            <td id="course-start-date">
+                @(Model?.Apprenticeships?.First().CourseStartDate.Value.ToString("MMMM yyyy"))                
```

Comment:
Please add CourseStartDateText property to the ApprenticeshipViewModel and set the format in the mapper.  Then it's just in one place.